src/Host/Client/Impl/Host/RHostConnector.cs

```diff
@@ -100,13 +106,12 @@ internal abstract class RHostConnector : IRHostConnector {
             }
         }
 
-        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, string rCommandLineArguments = null, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
+        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
```

Comment:
The change wrt `rCommandLineArguments` implies that every host process started from a given broker shares the same command line. While this is the way we're using it currently, it's not actually an inherent limitation on the other end - it's perfectly valid to spin up multiple hosts with different command lines - and it might come useful in the future.  So I would prefer the API that works with broker directly to not impose this limitation on its clients.